### PR TITLE
Added reponames filtering based on ARGV params

### PR DESCRIPTION
**Pull request by mdeniz**

### CODE REVIEW

File: pr_reminder.rb [hunk 1]
```diff
@@ -69,16 +74,12 @@ def pending_days
...
-end
-
-organization  = ARGV[0]
-
+organization = ARGV[0]
 repos = Repository.all(organization)
+if ARGV.size > 1
+  reponames = ARGV[1, ARGV.size]
+  repos.select! { |repo| reponames.include?(repo.name) }
+end
```
Comment (jreidinger) [inline pr_reminder.rb:78]: Well, I prefer more ruby code like

```
organization, *reponames = ARGV
repos = Repository.all(organization)
repos.select! { |repo| reponames.include?(repo.name) } unless reponames.empty?
```

which is three lines only and more redable for me.

Comment (jreidinger) [inline pr_reminder.rb:78]: btw `a, *b` syntax catch all possible cases we can have see this irb session demonstration it:

```
irb(main):013:0> repo, *names = [1]
=> [1]
irb(main):014:0> names
=> []
irb(main):015:0> repo, *names = []
=> []
irb(main):016:0> names
=> []
irb(main):017:0> repo, *names = [1,2,3,4]
=> [1, 2, 3, 4]
irb(main):018:0> names
=> [2, 3, 4]
```

File: pr_reminder.rb [hunk 2]
```diff
@@ -37,73 +57,45 @@ def any_pull_requests?
...
   def pending_days
-    ((Time.now - updated)/24/3600).floor
+    ((Time.now - updated) / 24 / 3600).floor
   end
 end
 
+def usage
+  <<-END
+    #{$PROGRAM_NAME} <github organization> [repo-name-1] [repo-name-2] ...  [repo-name-n]
+    Tool to print report about pending pull request of given organization
```
Comment (jreidinger) [inline pr_reminder.rb]: here should be explained behaviour if repo name is added

File: pr_reminder.rb [hunk 3]
```diff
@@ -1,35 +1,55 @@
...
+    pull_requests = []
+    url = "https://api.github.com/repos/#{full_name}/pulls"
+    open(url) do |json_file|
+      pull_requests = JSON.load(json_file.gets).map { |pr_data| PullRequest.new(pr_data) }
+    end
+    pull_requests.select! do |pull_request|
+      days = (Time.now.monday? || Time.now.tuesday?) ? 5 : 3 # do not count weekend in pending time
+      pull_request.pending_days > days
+    end
+    pull_requests
```
Comment (jreidinger) [inline pr_reminder.rb]: if you do not use `select!` and instead use `select` then this line is not needed as result of select is what we want.

Comment (mdeniz) [inline pr_reminder.rb]: Yes, added.

File: pr_reminder.rb (hunk 2 above)
Comment (mdeniz) [inline pr_reminder.rb]: Ok, I will.

**Comment by jreidinger** [comment]
thanks

**Pull request merged**